packages/tools/src/store/ToolGroupManager/ToolGroup.ts

```diff
@@ -185,6 +185,10 @@ export default class ToolGroup implements IToolGroup {
    * @param renderingEngineId - The rendering engine to use.
    */
   public addViewport(viewportId: string, renderingEngineId?: string): void {
+    if (typeof viewportId !== 'string') {
```

Comment:
How about !viewportId  since that checks for empty, null and undefined, which seems a better solution here.

Reply:
I was passing the `viewport` object itself, and I wasted 30 minutes on this trying to figure out what was wrong. So, `!viewportId` will not catch that, unfortunately.